gnu-linux/index.md

```diff
@@ -4,10 +4,10 @@ title: GNU/Linux
 ---
 
 # First Steps
-There are many different linux distros, but this guide is going to cover Ubuntu, which is one of the most common ones. The first thing you're going to need is the `build-essential` package, which contains the `gcc` and `g++` compilers, as well as the common build tool `make`. In order to do this, simply run (on the command line) `apt-get install build-essential`.   
+There are many different GNU/Linux distros, but this guide is going to cover Ubuntu, which is one of the more common ones. The first thing you're going to need is the `build-essential` package, which contains the `gcc` and `g++` compilers, as well as the common build tool `make`. In order to do this, simply run (on the command line) `apt-get install build-essential`.   
 
 # Editors
-If you're planning to write code, you're going to need a way to edit files! If you enjoy working on the command line, we'd recommend either `vim` or `nano`, while if you prefer a GUI, we recommend either [Atom](https://atom.io) or [Brackets](https://brackets.io).
+If you're planning to write code, you're going to need a way to edit files! The first thing you'll need to decide while picking an editor is whether you prefer a graphical editor (you interact using a mouse), or a keystroke based editor if you are very comfortable on the command line. Two of the most popular graphical editors are [Atom](https://atom.io) and [Brackets](https://brackets.io), while two of the most popular keystoke based editors are `vim` and `emacs`.
```

Comment:
We may potentially want to replace Brackets with Sublime Text, as Brackets is very heavily web focused